snap.html

```diff
@@ -25,6 +25,7 @@
 		<script type="text/javascript">
 			var world;
 			window.onload = function () {
+        SnapCloud = new Cloud(Cloud.determineCloudDomain());
```

Comment:
Because this is global, it seemed a little more obvious to be here than slightly hidden in cloud.js (since most other files don't create globals that are instances.)

Reply:
yeah, I wouldn't mind if it's in ```cloud.js```, in fact I might even prefer it there, keeping the current ````snap.html```` the way it is now, very concise with as little code as possible...